src/main/java/com/mastercard/creditanalytics/api/ApiExamples.java

```diff
@@ -145,37 +147,44 @@ public static MetricsPerLocation getFullyPopulatedMetrics() throws Unrecoverable
         );
     }
 
-    public static MetricsPerLocation getPartiallyPopulatedMetrics() throws UnrecoverableKeyException, CertificateException, IOException, KeyStoreException, NoSuchAlgorithmException, ApiException {
+    public static MetricsPerLocation getMerchantWithLowTransactionVolumeMetrics() throws UnrecoverableKeyException, CertificateException, IOException, KeyStoreException, NoSuchAlgorithmException, ApiException {
         return new MetricsApi(getApiClient()).getMetrics(
-                UUID.fromString(PARTIALLY_POPULATED_METRICS_LOCATION_ID),
+                UUID.fromString(MERCHANT_WITH_LOW_TRANSACTION_VOLUME_LOCATION_ID),
                 true
         );
     }
 
-    public static MetricsPerLocation getPartialYearMetrics() throws UnrecoverableKeyException, CertificateException, IOException, KeyStoreException, NoSuchAlgorithmException, ApiException {
+    public static MetricsPerLocation getMerchantWithNoDataFromCurrentOrPreviousYearYoyMetrics() throws UnrecoverableKeyException, CertificateException, IOException, KeyStoreException, NoSuchAlgorithmException, ApiException {
         return new MetricsApi(getApiClient()).getMetrics(
-                UUID.fromString(PARTIAL_YEAR_METRICS_LOCATION_ID),
+                UUID.fromString(MERCHANT_WITH_NO_DATA_FROM_CURRENT_OR_PREVIOUS_YEAR_YOY_LOCATION_ID),
+                true
+        );
+    }
+
+    public static MetricsPerLocation getMerchantWithLessThan52WeeksMetrics() throws UnrecoverableKeyException, CertificateException, IOException, KeyStoreException, NoSuchAlgorithmException, ApiException {
+        return new MetricsApi(getApiClient()).getMetrics(
+                UUID.fromString(NEW_MERCHANT_WITH_LESS_THAN_52_WEEKS_LOCATION_ID),
                 true
         );
     }
 
     public static MetricsPerLocation throwsMetricsNotFound() throws UnrecoverableKeyException, CertificateException, IOException, KeyStoreException, NoSuchAlgorithmException, ApiException {
         return new MetricsApi(getApiClient()).getMetrics(
-                UUID.fromString(NEWLY_ACCEPTING_LOCATION_ID),
+                UUID.fromString(MERCHANT_TOO_NEW_TO_HAVE_METRICS_LOCATION_ID),
                 true
         );
     }
 
     public static MetricsPerLocation throwsLocationNotFound() throws UnrecoverableKeyException, CertificateException, IOException, KeyStoreException, NoSuchAlgorithmException, ApiException {
         return new MetricsApi(getApiClient()).getMetrics(
-                UUID.fromString(NOT_FOUND_LOCATION_ID),
+                UUID.fromString("a1b2c3d4-0000-1234-abcd-000000000006"),
```

Comment:
Can we pull this out to a constant like the other location ids?

```suggestion
                UUID.fromString(MERCHANT_NOT_FOUND_LOCATION_ID),
```